packages/@react-spectrum/datepicker/docs/TimeField.mdx

```diff
@@ -0,0 +1,258 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/datepicker';
+import i18nDocs from 'docs:@internationalized/date';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/datepicker/package.json';
+
+```jsx import
+import {TimeField} from '@react-spectrum/datepicker';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Date and Time
+keywords: []
+after_version: 3.0.0
+---
+
+# TimeField
+
+<PageDescription>{docs.exports.TimeField.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['TimeField']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<TimeField label="Event time" />
+```
+
+## Value
+
+A `TimeField` displays a placeholder value by default. An initial, uncontrolled value can be provided to the `TimeField` using the `defaultValue` prop. Alternatively, a controlled value can be provided using the `value` prop.
+
+Time values are provided using objects in the `@internationalized/date` package. This library handles correct international date and time manipulation across calendars, time zones, and other localization concerns.
+
+`TimeField` only supports selecting times, but values with date components are also accepted. By default, `TimeField` will emit <TypeLink links={i18nDocs.links} type={i18nDocs.exports.Time} /> objects in the `onChange` event, but if a <TypeLink links={i18nDocs.links} type={i18nDocs.exports.CalendarDateTime} /> or <TypeLink links={i18nDocs.links} type={i18nDocs.exports.ZonedDateTime} /> object is passed as the `value` or `defaultValue`, values of that type will be emitted, changing only the time and preserving the date components.
+
+```tsx example
+import {Time} from '@internationalized/date';
+
+function Example() {
+  let [value, setValue] = React.useState(new Time(11, 45));
+
+  return (
+    <Flex gap="size-150" wrap>
+      <TimeField
```

Comment:
Is the TimeField supposed to change width when AM/PM changes?

Reply:
Not ideally. I can see about improving that, but in some locales that don't use similar width letters it might not be possible.